pkgs/memory_usage/README.md

```diff
@@ -0,0 +1,23 @@
+[![pub package](https://img.shields.io/pub/v/leak_tracker.svg)](https://pub.dev/packages/leak_tracker)
+[![package publisher](https://img.shields.io/pub/publisher/leak_tracker.svg)](https://pub.dev/packages/leak_tracker/publisher)
+
+Coming soon! See https://github.com/flutter/devtools/issues/3951 and https://github.com/flutter/devtools/issues/5606.
+
+## Status: Experimental
+
+**NOTE**: This package is currently experimental and published under the
+[labs.dart.dev](https://dart.dev/dart-team-packages) pub publisher in order to
+solicit feedback.
+
+For packages in the labs.dart.dev publisher we generally plan to either graduate
+the package into a supported publisher (dart.dev, tools.dart.dev) after a period
+of feedback and iteration, or discontinue the package. These packages have a
+much higher expected rate of API and breaking changes.
+
+Your feedback is valuable and will help us evolve this package. For general
+feedback, suggestions, and comments, please file an issue in the
+[bug tracker](https://github.com/dart-lang/leak_tracker/issues).
+
+## What is this?
```

Comment:
Keep in mind that visitors to the package on pub.dev will mainly see this readme. It's worth expanding the documentation here even if its duplicative of other docs (those more complete docs can then be referenced here).

Minimally, I would retain the 'memory usage tracking' section from https://pub.dev/packages/leak_tracker.